source/material_model/grain_size.cc

```diff
@@ -196,6 +196,24 @@ namespace aspect
 
 
 
+    template <int dim>
+    double
+    GrainSize<dim>::
+    compute_partitioning_fraction (const double temperature) const
+    {
+      const double power_term_numerator    =  std::pow (mantle_temperature, partitioning_exponent) -
+                                              std::pow (temperature, partitioning_exponent);
+
+      const double power_term_denominator  =  std::pow (mantle_temperature, partitioning_exponent) -
+                                              std::pow (surface_temperature, partitioning_exponent);
+
+      const double partitioning_fraction = min_f * (max_f/min_f) * (power_term_numerator/power_term_denominator);
```

Comment:
Shouldnt this be:
```suggestion
      const double partitioning_fraction = min_f * std::pow ((max_f/min_f), (power_term_numerator/power_term_denominator);
```
I am referring to equation 11 in Mulyukove & Bercovicy 2018, is that the same you were using?

Reply:
Yes, I forgot the power term. Thanks for catching this!